earth_enterprise/src/portableserver/build.py

```diff
@@ -147,12 +147,16 @@ def get_version(self):
                     if base_version and not base_version.startswith('#'):
                         break
             self.base_version = base_version
+            print ("Base version %s") % self.base_version
 
         if self.build_date is None:
             self.build_date = datetime.date.today()
-
+            print ("Date is %s") % self.build_date
+        print self.build_date.strftime('%Y%m%d')
+   
         if self.version_string is None:
-            self.version_string = '{}-{}'.format(
+            format_string = "{0}-{1}"
```

Comment:
Q: Is there a reason for preferring a variable containing a string literal, rather than using the string literal?